tests/merchStore.test.ts

```diff
@@ -343,10 +513,13 @@ describe('merch items with no options', () => {
   test('can delete all item options and add back options if the item is hidden', async () => {
     const conn = await DatabaseConnection.get();
     const admin = UserFactory.fake({ accessType: UserAccessType.ADMIN });
-    const item = MerchFactory.fakeItem({ hidden: true });
+    const collection = MerchFactory.fakeCollection();
```

Comment:
Is it necessary to fake a collection for testing item?

In other words, maybe we can look into rewriting the fakeitem to correctly create the new collection type.

Reply:
It is not, I believe I but that there to while trying to debug the error with the query.